Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/Broadcaster.cs

```diff
@@ -0,0 +1,193 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Extensions.Experimental.Socketer;
+using System;
+using System.IO;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    /// <summary>
+    /// This class observes changes and updates content on a user device.
+    /// </summary>
+    public class Broadcaster : Singleton<Broadcaster>
```

Comment:
>Broadcaster [](start = 17, length = 11)

Nit: Maybe SynchronizedStateBroadcaster? This name feels really generic as just Broadcaster.

Reply:
Or maybe BroadcasterNetworkManager, since this class deals with the networking necessary to send ComponentBroadcaster content across the network

---
In reply to: [284495745](https://github.com/microsoft/MixedRealityToolkit-Unity/pull/4386#discussion_r284495745) [](ancestors = 284495745)